### DISCUSSION TITLE
The staging branch has been created; release pushed to December 5th; Select PRs allowed to be submitted to staging

### DISCUSSION THREAD

**Discussion by BobbyRBruce** [gem5/gem5#1774] [Announcements]
Hey all,

The release staging branch has been created. However, unlike run-ups to previous gem5 releases, this one has had gem5 community members pushing relatively big features, some of which require more time for reviewing, testing, and development back-and-forth. While the original goal was to release v24.1 on November 22nd, doing so would require that the staging branch be ready for intensive testing now.

Given that these features need more time, we’ve decided to push the release to December 5th. While the staging branch has still been created, we will permit these PRs (listen below) to be merged within the next two weeks (as well as any bug fixes). In all cases, the contributors of these have been vocal about their desire to be part of the release and have worked with us to make this happen. Therefore, we’re willing to make an exception to our usual release procedures.

All other contributions to the develop branch (outside of bug fixes) will be assumed for 25.0 and can continue on as normally during this time.

We apologize for not giving ample notice of this release and have noted this as something to improve upon. We would, however, like to thank the great community development effort put into this release nonetheless.

- [x] #117 
- [x] #1084
- [x] #1433
- [x] #1636 
- [x] #1419 
- [x] #1743
- [x] #1534
- [x] #1489
- [x] #947 
- [x] #1768
- [ ] #1674
- [x] #1817 : Waiting for tests to pass then will merge.
- [ ] #1766 : Waiting for tests to pass then will merge. 
- [x] #1804
- [x] #1795 
- [x] #1813 
- [x] #1810 
- [x] #1491
- [x] #1829

**Comment by Harshil2107** [comment]
We should also fix this issue: https://github.com/gem5/gem5/issues/1674

**Comment by studyztp** [comment]
Can this bug fix also be in the staging branch? #1766

**Comment by BobbyRBruce** [comment]
#1804 should be in too

**Comment by giactra** [comment]
Hi Bobby, on which branch should I contribute RELEASE_NOTES.md updates?

**Reply by BobbyRBruce** [reply]
Honestly, either is fine for now. Though you can create PRs targeting the staging branch which may be slightly easier for me.

**Comment by erin-le** [comment]
PR  https://github.com/gem5/gem5/pull/1795 fixes the gem5/learning-gem5 daily test failure and should get in

**Comment by Harshil2107** [comment]
https://github.com/gem5/gem5/pull/1480 we should get this in as well to get the new gem5-bridge driver in

**Reply by giactra** [reply]
I don't want to call a single PR out but that PR has received limited reviews being still in draft state. IMHO we shouldn't rush new WIP features into the staging branch

**Reply by powerjg** [reply]
You're right, Giacomo. Apologies for not getting this ready for review sooner. We can drop it from the release.

**Comment by erin-le** [comment]
https://github.com/gem5/gem5/pull/1797 and https://github.com/gem5/gem5/pull/1810 are bug fixes that were submitted recently and haven't been reviewed. 

Giacomo's PR to fix the conflict between multi-Ruby and CHI-TLM: https://github.com/gem5/gem5/pull/1813 

Also, we previously discussed getting these config script updates in, but they aren't included above: https://github.com/gem5/gem5/pull/1491

**Comment by Harshil2107** [comment]
PR #1829 fixes the build failure when using `--duplicate-sources` after the new multiruby change. This also fixes the daily test build failure for sst and systemc tests

**Comment by erin-le** [comment]
#1817 and #1828 are approved bug fixes; putting them here for reference.

**Comment by BobbyRBruce** [comment]
Staging branch closed. V24.1 released: #1846
